tests/s3/versioning/test_versioning_delete_objects.py

```diff
@@ -0,0 +1,496 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""DELETE Objects test module for Object Versioning."""
+
+import logging
+import os
+import time
+import pytest
+
+from commons import error_messages as errmsg
+from commons.ct_fail_on import CTFailOn
+from commons.errorcodes import error_handler
+from commons.params import TEST_DATA_FOLDER
+from commons.utils.system_utils import create_file, path_exists, remove_file
+from commons.utils.system_utils import make_dirs, remove_dirs
+from commons.utils import assert_utils
+from config.s3 import S3_CFG
+from libs.s3.s3_test_lib import S3TestLib
+from libs.s3.s3_versioning_test_lib import S3VersioningTestLib
+from libs.s3.s3_multipart_test_lib import S3MultipartTestLib
+from libs.s3.s3_versioning_common_test_lib import check_get_head_object_version
+from libs.s3.s3_versioning_common_test_lib import check_list_object_versions
+from libs.s3.s3_versioning_common_test_lib import check_list_objects
+from libs.s3.s3_versioning_common_test_lib import delete_objects
+from libs.s3.s3_versioning_common_test_lib import delete_version
+from libs.s3.s3_versioning_common_test_lib import empty_versioned_bucket
+from libs.s3.s3_versioning_common_test_lib import upload_version
+from libs.s3.s3_versioning_common_test_lib import upload_versions
+
+
+class TestVersioningDeleteObjects:
+    """Test DELETE Objects API with Object Versioning"""
+
+    # pylint:disable=attribute-defined-outside-init
+    # pylint:disable-msg=too-many-instance-attributes
+    def setup_method(self):
+        """
```

Comment:
Docstring should be single liner if fit within line length limit (100 char). ex: """abc..."""

Reply:
done